scripts/retrieve_natura_raster.py

```diff
@@ -36,15 +36,17 @@
     if 'snakemake' not in globals():
         from _helpers import mock_snakemake
         snakemake = mock_snakemake('retrieve_natura_raster')
+        to_fn = Path("..", "resources/natura.tiff")
+    else:
+        to_fn = Path("resources/natura.tiff")
```

Comment:
What's wrong about `snakemake.output[0]` instead?

Reply:
You're too fast :)  See below

Reply:
sorry didn't read correctly, you are right